free-4.md

```diff
@@ -0,0 +1,80 @@
+# Ingredients to Dish
+
+## The user and a language
+This section describes who the project would serve and why a language might be a
+good way to meet their needs.
+
+This language seeks to provide a way for people to know what dish they can create based off of the ingredients they have. You can think of it as the quintessential cook book. The users will be able to choose from a variety of dishes they can actually make, rather than havin to query through several recipes they may not have the ingredients for. A language would be a good way to meet their needs as they could make the most out of their ingredients rather than settling for a dish missing ingredients.
+
+### What's the need?
+_What need is met by your idea? Who are you helping? What is that person's
+experience like now? What would their experience be like if you could help 
+them?_
+
+This idea makes it significantly easier to decide on what to make for a particular meal by eliminating most recipes. This idea helps people who don't have a large variety of ingredients to still make delicious dishes. As of now, the person either finds a recipe that they do not have all the ingredients for, spend a long time finding a recipe they actually have all the ingredients for, or completely forgo a recipe and try to throw together a dish. With this DSL, they could actually find usable recipes.
+
+### Why a language?
+_Why is a DSL appropriate for your user(s)? How does it address the need?_
+
+A DSL is appropriate for the users because deciding what to cook is a pretty specific domain. It addresses the need by enumerating all possible choices. 
+
+### Why you?
+_What excites you about this idea? How did you come up with it?_
+
+As someone who doesn't own a small nursery of herbs, it's disappointing to find a particularly appetizing recipe, only to realize it requires some obscure herb (I'm looking at you tarragon). I'm also realizing more just how important some of these missing ingredient can be (e.g. I didn't add ginger to a Chinese dish and it tasted super bland). It would be nice to be able to just see what I can actually make given my ingredients rather than waste time looking at every possible chicken recipe.
+
```

Comment:
I agree! I think there are some services in place that already have a lot of this functionality, so I'm curious how you would modify it to be different. I'll talk more about that in the appropriate section, though.